uh50_api/uh50.py

```diff
@@ -2,70 +2,300 @@
 This module reads from the Landys & Gyr UH50 heat measuring unit and returns the currect GJ and m3 meters.
```

Comment:
deze file doet te veel dingen
Er zit een main in, een functie om poorten op te zoeken en een class
die zou ik eerst eens opsplitsen in die 3 delen